docs/src/examples/components/Button/Types/index.tsx

```diff
@@ -14,6 +14,11 @@ const Types = () => (
       description="A button can be formatted to show different levels of emphasis."
       examplePath="components/Button/Types/ButtonExampleEmphasis"
     />
+    <ComponentExample
+      title="Secondary Alt"
+      description="Secondary Alt button."
```

Comment:
I would update this to say what the secondary alt button is for... like "button that inherits its background". I know that is very Teams theme specific, but I think this button is also very Teams specific..